bikestreets-ios/Map/MapsViewController.swift

```diff
@@ -7,15 +7,19 @@
 
 import MapboxMaps
 import MapboxSearch
+import MapboxNavigation
 import MapKit
 
 class MapsViewController: UIViewController {
-  private(set) lazy var mapView: MapView = {
-    return MapView(
-      frame: .zero,
-      mapInitOptions: .init(styleURI: currentMapboxStyle)
+  internal lazy var navigationMapView: NavigationMapView = {
+    return NavigationMapView(
+      frame: .zero
     )
   }()
```

Comment:
This can likely be simplified now, right?
```suggestion
  internal let navigationMapView = NavigationMapView(frame: .zero)
```